src/components/accordion-item/index.tsx

```diff
@@ -37,10 +38,12 @@ export class SmoothlyAccordionItem {
 			summary[0].addEventListener("click", onClick)
 			summary[0].addEventListener("touch", onClick)
 		}
-		this.smoothlyAccordionItemDidLoad.emit()
+		this.smoothlyAccordionItemWillLoad.emit(parent => {
+			this.parent = parent
+		})
 	}
 	disconnectedCallback() {
```

Comment:
due to how the smoothly app currently works any component that is placed as the main content will call this method even if we can see it. it seams to be working without the code inside of it.
![image](https://github.com/utily/smoothly/assets/25643315/78f8ab4f-b618-4053-9003-947a273b1459)
This picture is some logs from what happens when just loading the page. if you temp0 is acquired by right-clicking the array in the console -> store as global variable

Reply:
to clarify. if this is meant to handle clean-up it need to be done some other way. As of right now it might as well not exist unfortunately =/